recipe/meta.yaml

```diff
@@ -36,7 +36,8 @@ test:
 about:
   home: http://jupyter.org
   license: BSD 3-clause
-  summary: the base implementation of the Jupyter Notebook format
+  summary: Reference implementation of the Jupyter Notebook format
```

Comment:
I'll change it. Funny thing is, I took this from the github repo's description line.